src/main/webapp/about.jsp

```diff
@@ -16,13 +16,13 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>CodeU Chat App</title>
+  <title>Team int ellects; CodeU Chat App</title> <!-- added team name -->
   <link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
 
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a id="navTitle" href="/">Team int ellects; CodeU Chat App</a> <!-- added team name -->
```

Comment:
You don't really need these `<!-- added team name -->` comments, since GitHub will highlight your changes anyway. :cat: